modules/Evse15118D20/charger/session_logger.cpp

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+#include "session_logger.hpp"
+
+#include <chrono>
+#include <fstream>
+#include <iomanip>
+#include <stdexcept>
+
+#include <time.h>
+
+#include <date/date.h>
+
+#include <iso15118/session/logger.hpp>
+
+#include <everest/logging.hpp>
+
+using LogEvent = iso15118::session::logging::Event;
+
+std::string get_filename_for_current_time() {
+    const auto now = std::chrono::system_clock::now();
+    const auto now_t = std::chrono::system_clock::to_time_t(now);
+
+    std::tm now_tm;
+    localtime_r(&now_t, &now_tm);
```

Comment:
for logging, perhaps UTC would be better?
especially since the time zone isn't included in the returned string

Reply:
UTC is here better